beetsplug/lyrics.py

```diff
@@ -401,7 +401,13 @@ def _scrape_lyrics_from_html(self, html):
         # all of the lyrics can be found already correctly formatted
         # Sometimes, though, it packages the lyrics into separate divs, most
         # likely for easier ad placement
-        lyrics_div = soup.find("div", class_="lyrics")
+
+        lyrics_div = soup.find("div", {"data-lyrics-container": True})
+
+        if lyrics_div:
+            for br in lyrics_div.find_all("br"):
+                br.replace_with("\n")
```

Comment:
I'm not sure this is important enough to change, but we could reduce some duplication by factoring this out into a function (named `replace_br` or something?), which we'd also use below in the per-verse case.

Reply:
Just pushed a commit with a replace_br function to reduce code duplication